src/actions/index.js

```diff
@@ -375,12 +483,69 @@ export const usersDelete = (userName, auth_type) => {
   };
 }
 
-export const usersCreate = (user, type) => {
+export const groupDelete = (owner, name) => {
   return (dispatch) => {
     dispatch(loading(true))
-    agile.idm.user.create(user, type)
+    agile.idm.group.delete(owner, name)
+      .then(() => {
+        dispatch(action('GROUP_DELETE', name));
+        dispatch(message(`Group ${name} deleted.`));
+        dispatch(loading(false));
+      })
+      .catch(err => {
+        errorHandle(err, dispatch)
+      });
+  };
+}
+
+export const entityDelete = (entityId, entityType) => {
+  return (dispatch) => {
+    dispatch(loading(true))
+    agile.idm.entity.delete(entityId, entityType)
+      .then(() => {
+        dispatch(action('ENTITY_DELETE', entityId));
+        dispatch(message(`Entity ${entityId} deleted.`));
+        dispatch(loading(false));
+      })
+      .catch(err => {
+        errorHandle(err, dispatch)
+      });
+  }
+}
+
+export const entityDeleteByType = (entity, type) => {
```

Comment:
No, since this is just a function that is used for different types of entities and it delegates to other functions, which are actually doing the delete. The other functions are dispatching the event.